src-built-in/components/authentication/app.jsx

```diff
@@ -0,0 +1,84 @@
+/*!
+* The authentication component is a React component that renders a user form to collect auth info and determine whether to give user access to the rest of Finsemble
+* based on the information provided. 
+* Copyright 2020 by ChartIQ, Inc.
+* All rights reserved.
+*/
+
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import { AuthHeader, AuthForm, ErrorMessage, AuthInput, AuthSubmit } from "@chartiq/finsemble-ui/lib/components";
+import useAuth from "@chartiq/finsemble-ui/lib/hooks/useAuth";
+
+export const Authentication = () => {
+    const { saveInputChange, formValues, proceedToAuthorize, quitApplication } = useAuth();
+    const { error, setError } = useState(null);
+
+    /*
+     * Replace this function with your own authentication method that makes the call to your authentication server.
+     */
+    const authenticateUser = async () => {
+        // The user form data to be sent to the authentication server.
+        const { username, password } = formValues;
+        // Replace this with your own authentication calls. Currently it will always authenticate no matter what the user input is.
+        const result = await Promise.resolve({res: "ok"});
+        // the result is an object containing a response and an error. For example:
+        // {
+        //     res: "ok",
+        //     error: "Password is not correct"
+        // }
+        return result;
+    }
+
+    /**
+     * This function invokes the function that makes the call to the server that verifies user authentication information and
+     * decides whether to authorize the user to the rest of the application or display an error on the authentication form.
+     * @param {func} authenticateUser The function to call to authenticate the user.
+     */
+    const onSubmit = async (authenticateUser) => {
+        const authResult = await authenticateUser();
+        if (authResult.res === "ok") {
+            proceedToAuthorize();
+        } else {
+            setError(authResult.error);
+        }
+    };
+
+    return (
+        <>
+            <AuthHeader>
+                <img className="fsbl-logo" />
+                <div className="fsbl-close">
+                    <i className="ff-close" onClick={quitApplication}></i>
```

Comment:
Please try this out. I THINK the onClick should be on the parent div. I shouldn't have to click on the lines of the X in order to close the authentication component. I should be able to click on the padding around the x. If this doesn't make sense let me know.

Reply:
It doesn't matter. If you put the onclick handler on the icon the padding area will still detect onclick event. I think you were thinking of an `<svg>` situation